options/posix/generic/netdb-stubs.cpp

```diff
@@ -7,6 +7,13 @@
 #include <stdlib.h>
 #include <stddef.h>
 
+#undef h_errno
+int h_errno;
+
+int *__h_errno_location(void) {
+	return &h_errno;
+}
```

Comment:
Add a comment that this function is from glibc, it looks *really* strage (but the LSB standard contains it, so we should keep it).